test/Operators/finitedifference/column.jl

```diff
@@ -775,6 +775,196 @@ end
     end
 end
 
+
+@testset "Simple FCT: lin combination of UpwindBiasedProductC2F + Upwind3rdOrderBiasedProductC2F on (uniform) periodic mesh" begin
```

Comment:
I think these are already covered in the integration tests (some of which use quantitative tests to check against expected behavior). However, it would be good to improve some of our simple unit tests as well. But I don't think this PR is the right venue for that. We should discuss perhaps in a separate issue how to improve our unit tests. These are not the only one which need some improvement and/or corner cases covered.